tests/integration/aws/general/package/tests.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const execSync = require('child_process').execSync;
+const AWS = require('aws-sdk');
+const EOL = require('os').EOL;
+const fs = require('fs');
+
+const CF = new AWS.CloudFormation({ region: 'us-east-1' });
+const Utils = require('../../../../utils/index');
+
+describe('AWS - General: Deployment with --noDeploy', function () {
+  this.timeout(0);
+  let serviceName;
+  let deploy;
+
+  before(() => {
+    serviceName = Utils.createTestService('aws-nodejs', path.join(__dirname, 'service'));
+    deploy = execSync(`${Utils.serverlessExec} deploy --noDeploy`);
+  });
+
+  it('should deploy package with --noDeploy flag', () => {
+    const result = new Buffer(deploy, 'base64').toString();
+    const resultLines = result.split(EOL);
+    expect(resultLines[0]).to.equal('Serverless: Packaging service...');
```

Comment:
Should we still check for this one? IMHO the second check for `--noDeploy` is sufficient and less brittle